apps/src/applab/DesignModeBox.jsx

```diff
@@ -1,5 +1,3 @@
-/* global $ */
-
```

Comment:
There are a few cases, like this one, where jQuery was entirely unused in the file anyway.